desc/compute/_metric.py

```diff
@@ -1306,7 +1306,7 @@ def _g_sup_zz(params, transforms, profiles, data, **kwargs):
     coordinates="rtz",
     data=["e^rho", "e^theta"],
 )
-def _g_sup_rt(params, transforms, profiles, data, **kwargs):
```

Comment:
all the compute functions should also take `**kwargs` to maintain a consistent API

Reply:
Not sure what happened here to be honest - I don't remember deleting **kwargs but I'll fix (probably something dumb I was doing in the beginning)